content/en/docs/reference/glossary/hbone.md

```diff
@@ -0,0 +1,7 @@
+---
+title: HBONE
```

Comment:
Without a good definition, throwing new terms like HBONE is more confusing than clarifying IMHO.